src/NServiceBus.AcceptanceTests/Core/Sagas/When_saga_scanned_send_only_and_no_saga_storage.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace NServiceBus.AcceptanceTests.Core.Sagas
+{
+    using System;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_saga_scanned_send_only_and_no_saga_storage : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public void Should_not_throw()
+        {
+            Assert.DoesNotThrowAsync(async () =>
+            {
+                await Scenario.Define<Context>()
+                    .WithEndpoint<SendOnlyEndpointWithSaga>()
+                    .Done(c => c.EndpointsStarted)
+                    .Run();
+            });
+
+        }
+
+        class SendOnlyEndpointWithSaga : EndpointConfigurationBuilder
+        {
+            public SendOnlyEndpointWithSaga()
+            {
+                EndpointSetup<ServerWithNoDefaultPersistenceDefinitions>(c =>
+                {
+                    c.UsePersistence<InMemoryPersistence, StorageType.Subscriptions>();
```

Comment:
Is this because the inmemory storage tests included all tests including the core directory? (https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Persistence.InMemory.AcceptanceTests/NServiceBus.Persistence.InMemory.AcceptanceTests.csproj#L29)

Perhaps we should consider excluding it? (since we don't ship those tests to the downstream anyway) I can try that in a separate PR if you want?

Reply:
Yes the inmem Tests failed

I was thinking about excluding it but saw that we sind not exclude anything else so I figured this is ok